content/en/docs/tasks/traffic-management/ingress/ingress-sidecar-tls-termination/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: Ingress Sidecar TLS Termination
+description: Describes how to terminate TLS traffic at a sidecar without using an Ingress Gateway.
+weight: 30
+keywords: [traffic-management,ingress,https]
+owner: istio/wg-networking-maintainers
+test: yes
+---
+
+In a regular Istio mesh deployment, the TLS termination for downstream requests is performed at the Ingress Gateway.
+Though this satisfies most of the use cases, for some use cases (like an API Gateway on the mesh) the Ingress Gateway resource is not necessarily needed. This example describes how to eliminate the additional hop introduced by the Istio Ingress Gateway and let the Envoy sidecar running alongside the application perform TLS termination for incoming requests from outside the Istio service mesh.
+
+The example HTTPS service used for this task is a simple [httpbin](https://httpbin.org) service.
+In the following steps you first deploy an httpbin service inside your Istio service mesh with required configuration
+to perform TLS termination for downstream requests coming from outside the service mesh.
+
+{{< boilerplate experimental-feature-warning >}}
+
+## Before you begin
+
+*   Setup Istio by following the instructions in the [Installation guide](/docs/setup/), enabling the experimental
+    feature `ENABLE_TLS_ON_SIDECAR_INGRESS`.
+
+    {{< text bash >}}
+    $ istioctl install --set profile=demo --set meshConfig.outboundTrafficPolicy.mode=REGISTRY_ONLY --set values.pilot.env.ENABLE_TLS_ON_SIDECAR_INGRESS=true
+    {{< /text >}}
+
+*   Create the test namespace where the target `httpbin` service will be deployed. Make sure to enable sidecar injection
+    for the namespace.
+
+    {{< text bash >}}
+    $ kubectl create ns test
+    $ kubectl label namespace test istio-injection=enabled
+    {{< /text >}}
+
+## Enable global mTLS
+
+Peer Authentication Policy to allow mTLS traffic for all workloads has to be enabled.
+
+{{< text bash >}}
+$ kubectl -n test apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: default
+spec:
+  mtls:
+    mode: STRICT
+EOF
+{{< /text >}}
+
+## Disable PeerAuthentication for external mTLS Port
+
+The external mTLS Port that will be used at the sidecar for TLS Termination has to disable `PeerAuthentication`.
+
+{{< text bash >}}
+$ kubectl -n test apply -f - <<EOF
+apiVersion: security.istio.io/v1beta1
+kind: PeerAuthentication
+metadata:
+  name: disable-peer-auth-for-external-mtls-port
+  namespace: test
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+  mtls:
+    mode: STRICT
+  portLevelMtls:
+    9080:
```

Comment:
So the port should be separated from ports which are used for in-mesh mtls communication, it is better to highlight it.

Reply:
Please also highlight that the 9080 is the "targetPort" to avoid any misunderstanding.

Reply:
Done